app/controllers/school_classes_controller.rb

```diff
@@ -0,0 +1,32 @@
+class SchoolClassesController < ApplicationController
+  def new
+		@school_class = SchoolClass.new
+	end
```

Comment:
Woah...

Reply:
Honestly not sure how this happened because it looks completely normal on my end.

Reply:
CRAZY.